### ISSUE TITLE
Publish to npm?

### ISSUE THREAD

**Issue by codekitchen** [aficustree/homebridge-alarmdecoder-platform#3]
Hi, thanks for sharing this, it's exactly what I was looking for. Are you planning to publish it to npm? I started writing my own plugin after I couldn't find any alarmdecoder homebridge plugin on npm, but fortunately only like an hour later I ran across a reddit post linking to this repo.

**Comment by aficustree** [comment]
Hiya, I went ahead and bumped the version but you can find it on [npm](https://www.npmjs.com/package/homebridge-alarmdecoder-platform)

**Comment by codekitchen** [comment]
Oh I see where I went wrong, the Homebridge README suggests searching npm with the homebridge-plugin keyword, so I searched for "homebridge-plugin alarmdecoder" and couldn't find anything. Then when I saw that your repo suggests installing straight from git rather than the published npm package, I assumed it must not be on npm at all. My bad. Maybe add the alarmdecoder keyword to npm? Thanks again!
